AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-details/booking-details.component.ts

```diff
@@ -305,7 +311,8 @@ export class BookingDetailsComponent implements OnInit, OnDestroy {
         if (this.telephoneConferenceId && this.conferencePhoneNumber) {
             this.booking.telephone_conference_id = this.telephoneConferenceId;
             this.hearing.TelephoneConferenceId = this.telephoneConferenceId;
-            this.phoneDetails = `${this.conferencePhoneNumber} (ID: ${this.telephoneConferenceId})`;
+            this.phoneDetails = `ENG: ${this.conferencePhoneNumber} (ID: ${this.telephoneConferenceId})
+CY: ${this.conferencePhoneNumberWelsh} (ID: ${this.telephoneConferenceId})`;
```

Comment:
Very nit picky but could it be consistent for if we are using 2 letter or 3 letter codes by changing ENG to EN or CY to CYM?